Multisig/UI/Safe Management/Add Safe/CreatePasscodeSuggestionViewController.xib

```diff
@@ -41,8 +41,8 @@
                             <nil key="highlightedColor"/>
                         </label>
                         <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="MkS-TR-Xqe">
-                            <rect key="frame" x="0.0" y="38.5" width="374" height="61"/>
-                            <string key="text">Protect your data with a passcode used to unlock the Safe and confirm transactions.
+                            <rect key="frame" x="0.0" y="38.5" width="374" height="81.5"/>
```

Comment:
should not it be Safe{Wallet} mobile app ?

Reply:
it is Safe{Wallet}